### PR TITLE
CaaSP 4.0 fixes after rebase on SLE 15

### PR DESCRIPTION
**Pull request by kravciak**
Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/1013

Local runs:
 - http://dhcp126.suse.cz/tests/11651 - QA cluster
 - http://dhcp126.suse.cz/tests/11646 - QAM cluster
 - http://dhcp126.suse.cz/tests/11631 - MicroOS-10G-disk
 - http://dhcp126.suse.cz/tests/11637 - MicroOS-rcshell

### CODE REVIEW

File: lib/main_common.pm [hunk 1]
```diff
@@ -854,7 +854,7 @@ sub load_inst_tests {
             loadtest "installation/sles4sap_product_installation_mode";
         }
         # Kubic doesn't have a partitioning step
-        if (!is_caasp('kubic')) {
+        unless (is_caasp) {
```
Comment (dzedro) [inline lib/main_common.pm]: This has nothing to do with `kubic` now ?

Comment (kravciak) [reply lib/main_common.pm]: is_caasp returns true for both caasp & kubic - kubic is just filter parameter
```perl
is_caasp = is_caasp('caasp') || is_caasp('kubic')
```

Comment (dzedro) [reply lib/main_common.pm]: ok

File: tests/caasp/yast_welcome.pm [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+use base "y2logsstep";
+use utils;
+use testapi;
+
+sub run {
+    my $timeout = 120;
+    if (get_var('BETA')) {
+        assert_screen 'inst-betawarning', $timeout;
+        send_key 'ret';
+        $timeout = 30;
```
[the remark this answers is not in the document]
Comment (kravciak) [reply tests/caasp/yast_welcome.pm]: Removed, restarted local runs, let's see :)

**Pull request merged**